### ISSUE TITLE
Angular flavor crashes after splash

### ISSUE THREAD

**Issue by rynop** [NativeScript/nativescript-angular#1975]
**Environment**
```
$ tns info
✔ Component nativescript has 6.1.0 version and is up to date.
✔ Component tns-core-modules has 6.1.0 version and is up to date.
✔ Component tns-android has 6.1.0 version and is up to date.
✔ Component tns-ios has 6.1.0 version and is up to date.

$ node --version
v10.16.0
```

**Describe the bug**

`master` (https://github.com/NativeScript/NativeScript/commit/ca02ff1aedd76a1384bdbbd34d6f1e10286bc356) of `tns-core-modules` causes app crash immediately after launch via `tns run`.  Splash screen shows, then immediate crash without any error in the logs.

**To Reproduce**

```
$ node --version
v10.16.0
$tns --version
6.1.0
$ git clone https://github.com/NativeScript/NativeScript
$ tns create v6test --ng --appid com.rynop.v6test
$ cd v6test
$ npm install --save ../NativeScript/tns-core-modules
+ tns-core-modules@6.2.0
$ grep tns-core package.json
    "tns-core-modules": "file:../NativeScript/tns-core-modules",
$ cd ../NativeScript && npm install
$ npm run tsc-w
```

_In another tab:_

```
$ cd projects/v6test
$ tns run ios
```

**Expected behavior**

simulator/emulator should start and run without error.

**Additional context**

Prior to `6.1.0` this issue manifested itself in the following error:

> CONSOLE ERROR file:///src/Users/ryan/projects/NativeScript/tns-core-modules/trace/trace.ts:184:30 Error: Could not load CSS from app.css: TypeError: Expected module identifier to be a string.

Issue originally spotted in a somewhat related issue https://github.com/NativeScript/NativeScript/issues/7745 . Multiple people reporting this 
 same issue.

Not being able to run without webpack, makes it really difficult to track down and debug these type of startup/core NS problems.

**Comment by vpiskunov** [comment]
@rynop Does it actually crash fully for you? For me, I get the `Could not load CSS` error, but the app works fine...

**Comment by rynop** [comment]
Yup it does.  Prior to 6.1.0 I would get the CSS error and it would crash for me as well.

If there are any other environment / simulator / os values I can dump to help track down what the issue is, I'd be happy to post.

**Comment by vpiskunov** [comment]
@ryanp try using latest node version on LTS? Even just for the sake of just a different environment / clean install of tns

**Comment by rynop** [comment]
```
~/p/NativeScript (master|✔) $ rm -rf node_modules
~/p/NativeScript (master|✔) $ node --version
v10.16.3
~/p/NativeScript (master|✔) $ git pull
~/p/NativeScript (master|✔) $ npm install && npm run tsc-w

//in second tab:

~/p/v6test2 $ rm -rf platforms node_modules
~/p/v6test2 $ tns --version
6.1.0
~/p/v6test2 $ node --version
v10.16.3
~/p/v6test2 $ cat package.json | grep -i module
    "tns-core-modules": "file:../NativeScript/tns-core-modules",
~/p/v6test2 $ tns run ios
```

Launch, splash then crash again.

Full `tns run` log [here](https://gist.github.com/rynop/5156d00932b17937cc8889f2a0581f03)

package-lock.json [here](https://gist.github.com/rynop/874fdc601552547a7618b96f13f80b11)

**Comment by vpiskunov** [comment]
I’m suspicious on your double:
CONSOLE INFO file:///node_modules/nativescript-dev-webpack/hot.js:3:0: HMR: Hot Module Replacement Enabled. Waiting for signal.
CONSOLE INFO file:///node_modules/nativescript-dev-webpack/hot.js:3:0: HMR: Hot Module Replacement Enabled. Waiting for signal.

That shouldn’t happen. See my other issue here 
https://github.com/nativescript-vue/vue-cli-plugin-nativescript-vue/issues/16#issuecomment-496893226

Regarding the error 
`CONSOLE ERROR file:///src/Users/ryan/projects/NativeScript/tns-core-modules/trace/trace.ts:184:30 Error: Could not load CSS from app.css: TypeError: Expected module identifier to be a string.`
- so you don’t get it anymore? What caused it to disappear? 

Using Vue, not Angular, but seems it’s NS related

**Comment by rynop** [comment]
Your workaround in the comment your ref is for Vue (as you mentioned), not sure there is an equivalent workaround for angular.

I can't tell exactly what caused it to appear.  The only change from when I first hit this issue was upgrade to v6.1.0

If you take a look at the recent comments in https://github.com/NativeScript/NativeScript/issues/7745 you'll notice others are still getting the css error.

**Comment by elena-p** [comment]
@rynop, @vpiskunov,

What is the current version of `nativescript-dev-webpack` in your apps? We have seen this error, when you are using an older version of that dependency. With each {N} release, `nativescript-dev-webpack` should be updated as well, it is now being updated by the [`tns update` command](https://docs.nativescript.org/releases/upgrade-instructions#upgrading-the-application). With `tns-core-modules` 6.1 you should use `nativescript-dev-webpack` 1.2

**Comment by rynop** [comment]
Hi @elena-p mine is `1.2.0`.  I included my full `package-lock.json` in my previous post if you need to know other versions in my env.

@vpiskunov maybe `nativescript-dev-webpack` is what fixed the css err, but not the crash obv.
